src/lib/libmints/molecule.cc

```diff
@@ -245,6 +234,18 @@ Molecule& Molecule::operator=(const Molecule& other)
         atoms_.push_back((*iter)->clone(atoms_, geometry_variables_));
     }
 
+
+    // These are symmetry related variables, and are filled in by the following functions
+    pg_             = boost::shared_ptr<PointGroup>(new PointGroup(other.pg_));
```

Comment:
It may have been because `update_geometry()` called at the end of the constructor recomputes the point group, so it only needed a slot. I've no reason to think that copying then overwriting is bad, though.